components/lib/crash/src/main/java/mozilla/components/lib/crash/service/CrashReporterService.kt

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.crash.service
+
+import mozilla.components.lib.crash.crash.Crash
```

Comment:
`crash.crash.Crash` seems like one `crash` too many. Let's move the `Crash` class to the root package so it's `lib.crash.Crash`?

Reply:
Oh yeah, that make sense. In the IDE this looked fine because it only showed the sub tree. :)